pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -115,7 +115,7 @@ func init() {
 				// BuildDeleteController.PodManager (ControllerClient)
 				// BuildControllerFactory.buildDeleteLW
 				{
-					Verbs:     sets.NewString("get", "list", "create", "delete"),
+					Verbs:     sets.NewString("get", "list", "create", "delete", "deletecollection"),
```

Comment:
These don't actually use these powers yet, right?  Seems like we'd keep the list tight and make them update it when they use it.